packages/core/components/HeaderRibbon/index.tsx

```diff
@@ -17,34 +19,55 @@ const COLLECTION_TOOLTIP =
     'A "collection" is a previously saved set of files. Selecting one of these will narrow down the possible set of findable files down to those saved within this collection. You can create collections by selecting files, right-clicking, and selecting "Share Collection."';
 const VIEW_TOOLTIP =
     'A "view" is a pre-selected set of filters and/or sorts that when selected using this dropdown will add to (or potentially replace) your current filters and/or sorts with the pre-selected filters/sorts. You can then add or remove any filters/sorts at will.';
+const HELP_TOOLTIP =
+    "This menu provides quick links to related pages, as well as a tutorial system";
```

Comment:
nit The tool tips above end in a `.` but this one does not.  This should be consistent right?

Reply:
I just dislike a one sentence tooltip with punctuation personally. Technically they are meant to be small enough to not have any punctuation but... that didn't happen.